src/workerd/jsg/setup.h

```diff
@@ -310,7 +310,7 @@ class Isolate: public IsolateBase {
     // constructing a `Lock` on the stack.
 
   public:
-    Lock(const Isolate& isolate)
+    Lock(const Isolate& isolate, V8StackScope& scope)
```

Comment:
I'm assuming that this is passed here only to enforce the rule that it must exist before taking the lock? If so, a comment to that effect here would be helpful to warn off anyone who might come into this later and mistakenly assume that `scope` should be removed because it is unused here.